libs/storyboard-visualization/src/RoutesGraph/RoutesPreview.tsx

```diff
@@ -128,7 +128,7 @@ export function RoutesPreview(props: RoutesPreviewProps): React.ReactElement {
   };
 
   return (
-    <DndProvider backend={HTML5Backend} context={window}>
+    <DndProvider backend={HTML5Backend}>
```

Comment:
Remove `options.context` to use global instance to avoid the error of `cannot have two HTML5 backends at the same time`.

See https://github.com/react-dnd/react-dnd/blob/1fdd144f5e1bc723b9ac7c232b26892a16156f78/packages/react-dnd/src/core/DndProvider.tsx#L63